synapse/rest/client/report_event.py

```diff
@@ -62,12 +62,18 @@ async def on_POST(
                 Codes.BAD_JSON,
             )
 
-        event = await self._event_handler.get_event(
-            requester.user, room_id, event_id, show_redacted=False
-        )
+        try:
+            event = await self._event_handler.get_event(
+                requester.user, room_id, event_id, show_redacted=False
+            )
+        except AuthError:
+            # The event exists, but this user is not allowed to access this event.
+            event = None
+
         if event is None:
             raise NotFoundError(
-                "Unable to report event: it does not exist or you aren't able to see it."
+                "Unable to report event: "
+                "it does not exist or you aren't able to see it."
```

Comment:
I'm failing to spot a change here, was this your editor just wrapping it automatically or something?

Reply:
Oh yes. This was just me wrapping it as it went one whole character over 80, and that was bugging me.

Reply:
I usually let black handle this -- note that we have it configured to wrap at 88 characters, not 80.